workflow/scripts/de_analysis.py

```
@@ -155,39 +179,47 @@
-    cmap=snakemake.config["colormap"],
+    sorted.fillna(0).iloc[:n],
+    cmap=snakemake.config["deseq2"]["colormap"],
     linewidths=0,
     norm=LogNorm(),
 )
 plt.savefig(snakemake.output.de_top_heatmap)
 
 # our test case has no significant values
 # in our CI test, we have no significant data, hence:
-if snakemake.config["alpha"] < 0.9:
+if snakemake.config["deseq2"]["alpha"] < 0.9:
```

Ruff

Undefined name `snakemake`

(F821)